### PR TITLE
fix(Input): sanitized mask value no longer overwriting UI

### PR DESCRIPTION
**Pull request by benjamincharity**
ISSUES CLOSED: #1167 

Note: There is still an outstanding mask issue with percentages: https://github.com/GetTerminus/terminus-ui/issues/1371

### CODE REVIEW

File: terminus-ui/input/src/input.component.ts [hunk 1]
```diff
@@ -1427,7 +1427,6 @@ export class TsInputComponent implements
     } else {
       const finalValue = !this.maskSanitizeValue ? value : this.cleanValue(value, this.currentMask.unmaskRegex);
       this.onChangeCallback(finalValue);
-      this.inputValueAccessor.value = finalValue;
```
Comment (atlwendy) [inline terminus-ui/input/src/input.component.ts]: why would this remove the % symbol?

Comment (benjamincharity) [reply terminus-ui/input/src/input.component.ts]: From my current understanding, assigning back to the CVA here triggers Angular's internal updates which then overwrite the UI value with the model value.

Comment (shani-terminus) [reply terminus-ui/input/src/input.component.ts]: Weird

**Pull request merged**